lms/static/js/learner_dashboard/models/course_enroll_model.js

```diff
@@ -0,0 +1,20 @@
+/**
+ *  Track optIn status and submit form
+ */
+;(function (define) {
+    'use strict';
+
+    define([
+            'backbone'
+        ],
+        function( Backbone) {
+            return Backbone.Model.extend({
+                defaults: {
+                    course_id: '',
+                    optIn: false,
+                },
+                urlRoot: '/api/commerce/v0/baskets/'
```

Comment:
Why are comfortable hardcoding this URL, but not the others? The URL passing from the backed isn't really required as part of MVP if we're okay doing this.